studio/components/ui/PageTelemetry.tsx

```diff
@@ -54,6 +54,13 @@ const PageTelemetry: FC = ({ children }) => {
           language: telemetryProps?.language,
         },
       })
+
+      post(`${API_URL}/telemetry/pageview`, {
```

Comment:
Just double-checking -we want a call to `page` and to `pageview`?

Reply:
Yes pageview is a separate controller just for mixpanel